src/app/app.module.ts

```diff
@@ -1,28 +1,38 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
+import { JwtModule } from '@auth0/angular-jwt';
 
 import { AppComponent } from './app.component';
-import { HomePageComponent } from './components/home-page/home-page.component';
-import {ButtonModule} from "primeng/button";
-import { RegistrationComponent } from './components/registration/registration.component';
-import { ReactiveFormsModule } from '@angular/forms';
-import { LoginComponent } from './components/login/login.component';
 import { HttpClientModule } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { AuthService } from './public/services';
+import { PublicModule } from './public/public.module';
+import { ArticleModule } from './article/article.module';
 
 @NgModule({
   declarations: [
     AppComponent,
-    HomePageComponent,
-    RegistrationComponent,
-    LoginComponent
   ],
   imports: [
     AppRoutingModule,
     BrowserModule,
-    ButtonModule,
-    ReactiveFormsModule,
-    HttpClientModule
+    HttpClientModule,
+    PublicModule,
+    ArticleModule,
+    JwtModule.forRoot({
```

Comment:
No need to add `@auth0/angular-jwt` package, you can achieve the same result with `auth.service` + interceptors